src/fluree/json_ld/impl/expand.cljc

```diff
@@ -102,8 +102,9 @@
                     (contains? m :index))))))
 
 
-(defmulti parse-node-val (fn [v _ _ _ idx]
+(defmulti parse-node-val (fn [v v-info _ _ idx]
                            (cond
+                             (= :json (:type v-info)) :json
```

Comment:
`@json` will not be common, so in the interest of speed it would be ideal to include it at the bottom of evaluations unless it is needed to simplify something farther down the chain. Here the common scenarios now are now pushed below an additional 'if then' condition.

If using @type, then `v` will be a map. We do pick out @type in the `:map` condition... why not put this logic there?

Reply:
`v` could also be an array or really any js scalar value: boolean, number, string, null, etc. Mostly I don't think it would make sense to actually use js scalars as the value, but it's allowed.

And that's the reason I put it at the top of the cond list, really anything could be present in the `v` and still be `@type` `@json`.

Reply:
That makes sense, didn't think about it just being a scalar but still defined in the context as `@json`